packages/just-scripts-utils/src/mergePackageJson.ts

```diff
@@ -40,3 +39,29 @@ export function mergePackageJson(original: IPackageJson, incoming: IPackageJson)
 
   return packageJsonModified ? newPackageJson : original;
 }
+
+/**
+ * Exported for testing only. Determines whether a dep should be updated.
+ * Returns false if either version includes semver elements we don't understand
+ * (any characters besides number and ~ ^).
+ * @param originalVersion The original dep version (default 0.0.0)
+ * @param incomingVersion The incoming dep version
+ */
+export function _shouldUpdateDep(
```

Comment:
I like this convention of a "private" exported function using _ to start functions.